.github/workflows/ci.yml

```diff
@@ -257,9 +257,6 @@ jobs:
       - run: python -m pytest modin/test/storage_formats/base/test_internals.py
       - run: python -m pytest modin/test/storage_formats/pandas/test_internals.py
       - run: python -m pytest modin/test/test_envvar_npartitions.py
-      # TODO(https://github.com/modin-project/modin/issues/4550): this test is
-      # slow (~140 sec), but using 2 cores usually causes ray worker crashes.
-      # Fix the crashes and increase cores to 2.
       - run: python -m pytest modin/test/test_partition_api.py
```

Comment:
Should we return back pytest -n 2 for modin/test/test_partition_api.py?

Reply:
The test is so fast now that `-n 2` seems to actually slow the test down. on my mac, I get 24.07 sec with `-n 2`, and 13.3 sec with the default of 1. I'm on 

- macOSMonterey 12.2.1
- 2.3 GHz 8-Core Intel Core i9
- 16 GB 2667 MHz DDR4

Reply:
Ok, but your machine is different from those machines that are used in GH actions. It seems to me those have 2 CPUs each. I would suggest re-checking the need of `pytest -n 2 usage` for tests to understand if we really see the benefit. Of course, as part of a different issue.

Reply:
I agree with @YarShev that it's need to definne the need of using 2 parallel jobs with Ray/Dask clusters on 2 cores GH servers. I was also faced with issues when `-n 2` only increase execution time.